### PR TITLE
Make area-owners.json true source for all area like labels dependencies

### PR DESCRIPTION
**Pull request by marek-safar**

### CODE REVIEW

**Review by jeffschwMSFT — APPROVED** [review]
The owner updates look good to me.  @jeffhandley for the fabricbot updates.

File: docs/area-owners.md [hunk 1]
```diff
@@ -100,7 +104,7 @@ Note: Editing this file doesn't update the mapping used by `@msftbot` for area-s
...
 | area-System.Linq.Expressions                   | @jaredpar            | @cston @333fred                                     | Archived component - limited churn/contributions (see [#27790](https://github.com/dotnet/runtime/issues/27790))                                                                                                                                                                           |
-| area-System.Linq.Parallel                      | @jeffhandley         | @dotnet/area-system-linq-parallel                   | Consultants: @stephentoub @kouvel                                                                                                                                                                                                                                                         |
```
Comment (jeffhandley) [inline docs/area-owners.md:103]: We're losing "Consultants" notes in a few rows, but I think that's OK.

File: docs/area-owners.md [hunk 2]
```diff
@@ -2,9 +2,13 @@
 
 If you need to tag folks on an issue or PR, you will generally want to tag the owners (not the lead) for [area](#areas) to which the change or issue is closest to. For areas which are large and can be operating system or architecture specific it's better to tag owners of [OS](#operating-systems) or [Architecture](#architectures).
 
-## Areas
+## Subsciption and Notifications
+
+The tables bellow are generated and for informational purposes only. If you would like to be included or removed from area-specific issue/PR notifications submit PR with update to data source files. There are two ways how to do that
+* Manually update `mentionees` sections in both [`fabricbot.json`](fabricbot.json) and [`.github/fabricbot.json`](../.github/fabricbot.json) files
```
Comment (jeffhandley) [inline docs/area-owners.md:8]: I suspect you meant to refer to `area-owners.json` and `fabricbot.json` here.

**Review by jeffhandley — CHANGES_REQUESTED** [review]
I left comments in https://github.com/dotnet/fabricbot-config/pull/69#pullrequestreview-1330436620, but this is currently resulting in an undesired change to mentionees in the issue routing where `@dotnet/area-` team members are expanded and mentioned specifically as well.

_Thanks for your patience awaiting my review_.

**Pull request closed without merging**